assemblies/assembly-business-optimizer-springboot.adoc

```diff
@@ -59,6 +59,43 @@ include::{enterprise-dir}/optimizer/business-optimizer-logging-proc.adoc[levelof
 
 include::{enterprise-dir}/optimizer/business-optimizer-database-UI-integration-proc.adoc[leveloffset=+1]
 
+== Using Micrometer and Prometheus to monitor your school timetable OptaPlanner Spring Boot application
+
+OptaPlanner exposes metrics through https://micrometer.io/[Micrometer], a metrics instrumentation library for Java applications. You can use Micrometer with Prometheus to monitor the OptaPlanner solver in the school timetable application.
+
+
+.Prerequisites
+* You have created the Spring Boot OptaPlanner school timetable application.
```

Comment:
I would add here that user should have Prometheus installed as well. Do we have some instruction how to do that in product doc?

Reply:
Not for installing. I directed them to the Prometheus site.

Reply:
Updated

Reply:
LGTM

Reply:
LGTM